mriqc/interfaces/bids.py

```diff
@@ -224,6 +234,12 @@ def get_metadata_for_nifti(in_file):
     side_json = fname + '.json'
     fname_comps = op.basename(side_json).split("_")
 
+    #just for an other json file convention (should not interfer) but can be removed
```

Comment:
If you protect this function from being called when there's no BIDS structure at the input, then it is better not to preemptively check inside. Otherwise, bugs will be really hard to chase. This function should not be modified.